hw2/05-form.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title>05 - Form</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+  <script defer src="05-form.js"></script>
+</head>
+
+<body>
+  <main>
+    <div class="jumbotron d-flex align-items-center bg-dark w-100 vh-100 px-2">
+      <main class="container col-lg-5 col-md-6 col-sm-8 bg-light">
+        <form class="col-12">
+          <h1 class="my-4 text-center">Exercise 05 - Form</h1>
+          <div class="row m-1">
+            <label for="name" class="col-md-4 fw-bold">Full Name:</label>
+          </div>
+          <div class="row mx-2 my-2">
+              <input type="text" id="name" name="name" class="form-control" />
```

Comment:
could always add the required attribute. so that they can't just submit an empty form. that or add checks to the js side

Reply:
I already check to see if there is no submission in my javascript and output 'No Submission' if so.